modules/database-mongo/README.md

```diff
@@ -0,0 +1,37 @@
+## Agent configuration
+
+This module uses [collectd/mongodb](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-mongodb.html) monitor.
+
+```yaml
+  - type: collectd/mongodb
+    host: &mongoHost localhost
+    port: &mongoPort 27017
+    username: user
+    password: pass
+    databases:
+      - admin
+    # Uncomment only if mysql server is not on the same host as signalfx agent
+    #disableHostDimensions: true
+    extraDimensions:
+      # Uncomment only if you enabled `disableHostDimensions` or for "serverless" mode.
+      #host: *mongoHost
+    # You should not have to change lines below
+    extraMetrics:
+      - gauge.connections.available
+      - counter.asserts.regular
+      - counter.asserts.warning
+      # Only required if agent <= 5.5.5:
+      - gauge.repl.max_lag
+      - gauge.repl.active_nodes
+      - gauge.repl.is_primary_node
+```
+
+## Notes
+
+* Primary and secondary detectors require to configure on all members 
```

Comment:
I do not understand what you mean by "require to configure on all members". I think you need to rephrase the statement.